pkg/agent/qrm-plugins/cpu/dynamicpolicy/policy.go

```diff
@@ -1938,6 +1941,17 @@ func (p *DynamicPolicy) allocateByCPUAdvisorServerListAndWatchResp(resp *advisor
 }
 
 func (p *DynamicPolicy) serveCPUPluginCheckpoint(stopCh <-chan struct{}) {
+	cpuPluginSocketDir := path.Dir(p.cpuPluginSocketAbsPath)
```

Comment:
somehow we should find a way to extract those similar logic in a centralized function, i already add a new issue for this topic https://github.com/kubewharf/katalyst-core/issues/30,  just notify, no need to do it now